spec/system/entity_index_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
```

Comment:
- Hey, ALL YOUR TEST IN THE `spec/system` fails. Can you please, reevaluate to fix them?